frontend/src/app/components/admin/audit-pane/audit-pane.js

```diff
@@ -160,9 +160,8 @@ class AuditPaneViewModel extends ConnectableViewModel {
 
     constructor(...args) {
         super(...args);
-
         this.dispatch(fetchAuditLog(
-            { categories: Object.keys(categoryDisplayNames) },
+            { categories: Object.keys(categoryDisplayNames).filter(i => (i !== 'bucket' && i !== 'obj')) },
```

Comment:
@nimrod-becker  Please don't use var names such as `i`, `category` or 'name' may be more reasonable
Also, the line is too long (above 100 chars)

As a recommendation, in this case, it will be more readable to create a local var name categories outside the dispatch call